src/contour/Config.h

```diff
@@ -351,7 +351,7 @@ struct TerminalProfile
         statusDisplayPosition { vtbackend::StatusDisplayPosition::Bottom };
     ConfigEntry<std::string, documentation::IndicatorStatusLineLeft> indicatorStatusLineLeft {
         " {InputMode:Bold,Color=#FFFF00}"
-        "{Tabs:Left= │ }"
+        "{Tabs:ActiveColor=#FFFF00,Left= │ }"
```

Comment:
We need to change it in the documentation as well, and add ActiveBackground info

Reply:
I somehow didn t get the config values out of contour.yml, that is why I left it as draft and took a nap instead of finishing it.  I hope to figure out the reason today :)

Reply:
EDIT: loading config worked all night long. I obviously edited the wrong contour.yml and wondered why it wasn't reflecting my changes :D